//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// abcdefg_NO_VP.h
//
// Code generation for function 'abcdefg_NO_VP'
//

#ifndef ABCDEFG_NO_VP_H
#define ABCDEFG_NO_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdefg_NO_VP(double P_init, double V_init, double A_init, double P_wayp,
                   double V_wayp, double V_max, double V_min, double A_min,
                   double J_max, double J_min, double t[14]);

#endif
// End of code generation (abcdefg_NO_VP.h)
